### PR TITLE
feat: pagination in auditlog

### PR DESCRIPTION
**Pull request by bvektor**
feat: pagination hook
feat: paginator component
feat: pagination in auditlog
refactor: dyo-list headers

### CODE REVIEW

File: web/crux-ui/src/pages/teams/audit-log.tsx [hunk 1]
```diff
@@ -58,18 +60,29 @@ const AuditLogPage = (props: AuditLogPageProps) => {
...
     url: ROUTE_TEAMS_AUDIT,
   }
 
-  const listHeaders = ['common:name', 'common:date', 'event', 'info'].map(it => t(it))
+  const pagination = usePagination<AuditLog>({
+    initialData: auditLog,
+    initialPagination: { pageSize: 10, currentPage: 0 },
+  })
+
+  const listHeaders = ['', 'common:name', 'common:date', 'event', 'info'].map(it => t(it))
```
Comment (polaroi8d) [inline web/crux-ui/src/pages/teams/audit-log.tsx:77]: Why we need an empty item in the array?

**Review by polaroi8d — COMMENTED** [review]
Please pay attention to squash the comments. LGTM (Informal) 

@m8vago please review.

File: web/crux-ui/src/pages/teams/audit-log.tsx (hunk 1 above)
Comment (bvektor) [reply web/crux-ui/src/pages/teams/audit-log.tsx:77]: Because the first column (the placeholder avatar) doesn't have a header text just an empty cell.

**Review by robot9706 — APPROVED** [review]
🚀🚀🚀

File: web/crux-ui/src/elements/dyo-list.tsx [hunk 2]
```diff
@@ -31,55 +33,57 @@ export const DyoList = <T,>(props: DyoListProps<T>) => {
...
+                  </div>
+                </div>
+              ))}
+            </div>
+          </div>
+        ) : null}
+        <div className="table-row-group">
+          {data.map((line, rowIndex) => (
+            <div className="table-row" key={`${props.key}-${rowIndex}`}>
+              {line.map((_, column) => (
```
Comment (m8vago) [inline web/crux-ui/src/elements/dyo-list.tsx:70]: could we use `row` instead of `line`?

Comment (m8vago) [reply web/crux-ui/src/elements/dyo-list.tsx:70]: and column should be `colIndex` i guess

**Review by m8vago — APPROVED** [review]

**Pull request merged**